### PR TITLE
add position-percentage as a statusline indicator

### PR DESCRIPTION
**Pull request by sbromberger**
This PR adds a(n optional) percentage indicator to statusline. Inspired by https://github.com/helix-editor/helix/discussions/3156#discussion-4238768.

### CODE REVIEW

File: book/src/configuration.md [hunk 1]
```diff
@@ -78,6 +78,7 @@ The following elements can be configured:
 | `diagnostics` | The number of warnings and/or errors |
 | `selections` | The number of active selections |
 | `position` | The cursor position |
+| `position-pct` | The cursor position as a percentage of the total number of lines |
```
Comment (sudormrfbin) [inline book/src/configuration.md:81]: `position-percentage` is more readable and saving characters doesn't do us any favors any way.

Comment (sbromberger) [reply book/src/configuration.md]: renamed per suggestion. Thanks!

**Comment by archseer** [comment]
Can you resolve the conflicts? Looks fine to merge otherwise

**Comment by ibrahimqasim-abtrace** [comment]
There were no spaces around the value. Will raise an issue, should be an easy fix.

**Pull request merged**